Textile/Textile.sublime-syntax

```diff
@@ -1,225 +1,1126 @@
+    - include: merge-conflict-markers
+
   main:
-    - match: '(^h[1-6]([<>=()]+)?)(\([^)]*\)|\{[^}]*\})*(\.)'
+    - meta_prepend: true
+    - include: blocks
+    - include: inlines
+
+###[ HTML ]####################################################################
+
+  # https://textile-lang.com/doc/html
+
+  html:
+    - meta_prepend: true
+    - include: html-notextiles
+
+  html-notextiles:
+    - match: (<)(notextile)[ \f\t]*(>)
+      scope: meta.tag.notextile.begin.html
+      captures:
+        1: punctuation.definition.tag.begin.html
+        2: entity.name.tag.other.html
+        3: punctuation.definition.tag.end.html
+      push: html-notextile-body
+
+  html-notextile-body:
+    - meta_content_scope: text.plain
+    - match: (</)(notextile)[ \f\t]*(>)
+      scope: meta.tag.notextile.end.html
+      captures:
+        1: punctuation.definition.tag.begin.html
+        2: entity.name.tag.other.html
+        3: punctuation.definition.tag.end.html
+      pop: 1
+
+###[ BLOCKS ]##################################################################
+
+  # https://textile-lang.com/category/block-modifiers/
+
+  blocks:
+    - include: textile-comments         # ###.. or ###.
+    - include: code-blocks              # bc.. or bc.
+    - include: quotation-blocks         # bq.. or bq.
+    - include: notextile-blocks         # notextile.. or notextile.
+    - include: pre-blocks               # pre.. or pre.
+    - include: headings                 # h1. to h6.
+    - include: notes
+    - include: footnotes
+    - include: tables
+    - include: lists
+    - include: paragraphs
+
+###[ BLOCKS / COMMENTS ]#######################################################
+
+  # https://textile-lang.com/doc/textile-comments
+  # NOTE: textile comments are just regular (extended) blocks
+
+  textile-comments:
+    - match: '^(###)(\.\.)[ ](?=\S)'
+      captures:
+        1: punctuation.definition.comment.textile
+        2: punctuation.definition.generic.extended-block.textile
+      push: extended-textile-comment-body
+    - match: '^(###)(\.)[ ](?=\S)'
+      captures:
+        1: punctuation.definition.comment.textile
+        2: punctuation.definition.generic.block.textile
+      push: textile-comment-body
+
+  extended-textile-comment-body:
+    - meta_scope: comment.block.textile
+    - include: next-block-pop
+
+  textile-comment-body:
+    - meta_scope: comment.block.textile
+    - include: empty-line-pop
+
+###[ BLOCKS / HEADINGS ]#######################################################
+
+  # https://textile-lang.com/doc/headings
+
+  # TODO: clean these up, not much different other than the beginning of the
+  #       line and the meta_scope
+
+  headings:
+    - match: |-
+        (?x:
+          ^
+          (h1)
+          ({{textile_indentation_optional}})
+          ({{textile_alignment_optional}})
+          ({{textile_attributes_optional_class_style}})
+          (\.)
+          [ ](?=\S)
+        )
```

Comment:
FWIW, a global (?x) would work here too, saving a line of code and a level of indentation.

But that's purly question of taste, not a request to change it as both stles are equal.

```
    - match: |-
        (?x)
        ^
        (h1)
        ({{textile_indentation_optional}})
        ({{textile_alignment_optional}})
        ({{textile_attributes_optional_class_style}})
        (\.)
        [ ](?=\S)
      captures:
        ...
```